cartridges/app_storefront_base/cartridge/templates/default/cart/cart.isml

```diff
@@ -20,20 +20,23 @@
     </div>
 
     <div class="container">
-        <h1 class="page-title">${Resource.msg('title.cart','cart',null)}</h1>
```

Comment:
Please, extend cart.isml and productDetails.isml in the custom cartridge. We don't want any changes to app_storefront_base

Reply:
done